TickSpec/TickSpec.fsproj

```diff
@@ -1,63 +1,19 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{97bf4912-2b4e-410b-9b63-5e07076514a9}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>TickSpec</RootNamespace>
-    <AssemblyName>TickSpec</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <Name>TickSpec</Name>
-    <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
-    <TargetFrameworkProfile />
-    <TargetFSharpCoreVersion>4.3.0.0</TargetFSharpCoreVersion>
+    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <Title>TickSpec</Title>
+    <Description>
```

Comment:
Is this used even when paket is used to create the nuget packages?

Reply:
Paket pack doesn't seem to deal well with multi-target projects (the resulting nuget contains one target only). It seems that dotnet pack is the advised solution, so I migrated to that. These elements are used.

Reply:
Ok, fine for me. The created package looks good.